ajax/libs/tent-css/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "tent-css",
+  "filename": "dist/tent.min.css",
```

Comment:
Path to main file should be relative path to `basePath` in auto-update config.
So here would be `tent.min.css`.